ajax/libs/vertx/package.json

```diff
@@ -17,6 +17,18 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/vert-x/vert.x"
+    "url": "git://github.com/vert-x3/vertx-bus-bower"
```

Comment:
This seem to be a release purpose repo, that's not what we expected, you already use it at the auto-update config, this part should be a development purpose one.